dbt/models/qc/schema.yml

```diff
@@ -823,7 +1061,8 @@ models:
         - qc_report_town_close
     meta:
       export_name: 0 land value
-      export_template: assessed_and_market_values
+      export_template:
+        name: assessed_and_market_values
```

Comment:
This interface has changed in the export script, so we have to migrate existing instances.